readme.md

```diff
@@ -1,8 +1,8 @@
 # Figstract
 
 [![Maven Central Version](https://img.shields.io/maven-central/v/com.anifichadia.figstract/cli)](https://central.sonatype.com/namespace/com.anifichadia.figstract)
-![Release](https://github.com/AniFichadia/Figstract/actions/workflows/release.yml/badge.svg)
-![Build](https://github.com/AniFichadia/Figstract/actions/workflows/build.yml/badge.svg)
+[![Release](https://github.com/AniFichadia/Figstract/actions/workflows/release.yml/badge.svg)](https://github.com/AniFichadia/Figstract/actions/workflows/release.yml)
+[![Build](https://github.com/AniFichadia/Figstract/actions/workflows/build.yml/badge.svg)](https://github.com/AniFichadia/Figstract/actions/workflows/build.yml)
```

Comment:
Force this to be the main branch
https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/monitoring-workflows/adding-a-workflow-status-badge#using-the-branch-parameter